### DISCUSSION TITLE
Static Web Site and Tab only projects

### DISCUSSION THREAD

**Discussion by wictorwilen** [pnp/generator-teams#294] [Ideas]
**Is your feature request related to a problem? Please describe.**
Make it easier to create and publish client side only apps.

**Describe the solution you'd like**
I'd like to see an option where **yo teams** scaffolds a project with ONLY tab support - and with the possibility to publish to a static web site. That is - not spinning up express or anything like that when in production.

**Comment by qu0cster** [comment]
I agree...just need the results of the client side.... I am irritated by how slow the "gulp serve" is.  To make it worse, it always crash and I have to restart it.  Don't know what its problem:

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 1: 00007FF706747DDA v8::internal::GCIdleTimeHandler::GCIdleTimeHandler+4506
 2: 00007FF706722876 node::MakeCallback+4534
 3: 00007FF7067231F0 node_module_register+2032
 4: 00007FF706A3B6BE v8::internal::FatalProcessOutOfMemory+846
 5: 00007FF706A3B5EF v8::internal::FatalProcessOutOfMemory+639
 6: 00007FF706C21DE4 v8::internal::Heap::MaxHeapGrowingFactor+9620
 7: 00007FF706C18DC6 v8::internal::ScavengeJob::operator=+24550
 8: 00007FF706C1741C v8::internal::ScavengeJob::operator=+17980
 9: 00007FF706C20167 v8::internal::Heap::MaxHeapGrowingFactor+2327
10: 00007FF706C201E6 v8::internal::Heap::MaxHeapGrowingFactor+2454
11: 00007FF706D4A487 v8::internal::Factory::NewFillerObject+55
12: 00007FF706DC7676 v8::internal::operator<<+73494
13: 000001ABE735C5C1

**Comment by wictorwilen** [comment]
What if we added support for running your tabs with `react-scripts`, but still scaffold the server side (which don't have to be used)?
